sys/net/bpf.h

```diff
@@ -70,7 +70,14 @@ typedef	u_int64_t bpf_u_int64;
  */
 struct bpf_program {
 	u_int bf_len;
-	struct bpf_insn * __kerncap bf_insns;
+#ifdef _KERNEL
+	union {
```

Comment:
This is a bit of a gross hack so that curly-brace initialisation works in userspace (though the explicit member dot syntax still works), and rarpd does just that. Otherwise even though it's unambiguous for a single-member union you still get a warning that becomes an error (`warning: suggest braces around initialization of subobject [-Wmissing-braces]`) with our compiler flags. Technically we should do the same for the existing CAM changes that add unions to preserve API compatibility in userspace (though one could argue it's a bad idea to use such constructs and you should use the dot syntax or the boring "create the object then assign each member" approach).